sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/tests/Azure.Monitor.OpenTelemetry.LiveMetrics.Demo/Program.cs

```diff
@@ -28,6 +30,17 @@ public static async Task Main(string[] args)
                             .AddLiveMetrics(configure => configure.ConnectionString = ConnectionString)
                             .Build();
 
+            using var loggerFactory = LoggerFactory.Create(builder =>
+            {
+                builder.AddOpenTelemetry(options =>
+                {
+                    //options.IncludeFormattedMessage = true;
+                    options.AddLiveMetrics(configure => configure.ConnectionString = ConnectionString);
```

Comment:
If this becomes internal, we don't need to parse options within the logging extensions. It should work in order where tracing initializes and logs utilizes it.